datafusion/expr/src/expr_fn.rs

```diff
@@ -731,7 +731,7 @@ scalar_expr!(
 scalar_expr!(
     ArraySlice,
     array_slice,
-    array offset length,
+    array offset length stride,
```

Comment:
I think we should change this to `array begin end step`. step to be consistent with duckdb docs